### PR TITLE
Add PostgreSQL ActiveRecord Sample

### LINKED ISSUE #471: Create Cloud SQL connectivity sample for Postgres
Create a copy of the [MySQL connectivity sample](https://github.com/GoogleCloudPlatform/ruby-docs-samples/tree/master/cloud-sql/mysql/activerecord) and update it to work with Postgres.

Requirements:

- PR the sample into this repo under `cloud_sql/postgres/activerecord`
- Update the following region tag(s): 
    * `cloud_sql_mysql_activerecord_connection` -> `cloud_sql_postgres_activerecord_connection`
    * `cloud_sql_mysql_activerecord_create` -> `cloud_sql_postgres_activerecord_create`
    * `cloud_sql_mysql_activerecord_limit` -> `cloud_sql_postgres_activerecord_limit`
    * `cloud_sql_mysql_activerecord_timeout` -> `cloud_sql_postgres_activerecord_timeout`
    * `cloud_sql_mysql_activerecord_backoff` -> `cloud_sql_postgres_activerecord_backoff`
- Update the README for Postgres and verify the instructions work

Please add myself as a reviewer.

Assigned: @quartzmo

FYI: @dmahugh, @jsimonweb

### PR DESCRIPTION
**Pull request by shubha-rajan**
Fixes #471 
Region tags are in `config/database.yml` and `app/controllers/votes_controller.rb`

### CODE REVIEW

**Comment by kurtisvg** [comment]
@TheRoyalTnetennba - Would you mind reviewing this from a Ruby perspective?

File: cloud-sql/postgres/activerecord/config/database.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  # [END cloud_sql_postgres_activerecord_backoff]
+  
+  # [END_EXCLUDE]
+  username: <%= ENV["PG_USERNAME"] %>
+  password: <%= ENV["PG_PASSWORD"] %>
+  database: <%= ENV.fetch("PG_DATABASE") { "vote_development" } %>
+  # If connecting via unix domain socket, specify the socket path as host
+  host:  '/cloudsql/<%= ENV["INSTANCE_CONNECTION_NAME"] %>'
+  # If connecting via TCP, enter the IP and port instead
+  # host: '127.0.0.1'
```
Comment (kurtisvg) [inline cloud-sql/postgres/activerecord/config/database.yml:42]: I know we use a different IP down below, but we should probably clarify here as well. User's will only see this portion from c.g.c.

File: cloud-sql/postgres/activerecord/README.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+[instructions](https://cloud.google.com/sql/docs/mysql/connect-external-app#4_if_required_by_your_authentication_method_create_a_service_account).
+Download a JSON key to use to authenticate your connection. 
+
+1. Use the information noted in the previous steps:
+```bash
+export GOOGLE_APPLICATION_CREDENTIALS=/path/to/service/account/key.json
+export INSTANCE_CONNECTION_NAME='<MY-PROJECT>:<INSTANCE-REGION>:<INSTANCE-NAME>'
+export PG_USERNAME='my-db-user'
+export PG_PASSWORD='my-db-pass'
+export PG_DATABASE='my_db'
```
[the remark this answers is not in the document]
Comment (shubha-rajan) [reply cloud-sql/postgres/activerecord/README.md]: Changed in latest commit. We should change these in the existing mysql sample too.

File: cloud-sql/postgres/activerecord/README.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+Finally, start the application:
+```bash
+bundle exec rails s
+```
+
+Navigate towards `http://localhost:3000` to verify your application is running correctly.
+
+## Google App Engine Flexible
+
```
[the remark this answers is not in the document]
Comment (shubha-rajan) [reply cloud-sql/postgres/activerecord/README.md:97]: Added instructions in the last commit. The mysql sample is also missing Standard instructions.

File: cloud-sql/postgres/activerecord/app.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Copyright 2019 Google, Inc
```
[the remark this answers is not in the document]
Comment (shubha-rajan) [reply cloud-sql/postgres/activerecord/app.yaml]: updated.

File: cloud-sql/postgres/activerecord/config/database.yml (hunk 1 above)
Comment (shubha-rajan) [reply cloud-sql/postgres/activerecord/config/database.yml]: fixed

**Pull request merged**